### PR TITLE
Wrap urlopen() exception with SwaggerValidationError, call urlopen() once instead of two times.

### LINKED ISSUE #6: All the urlopen calls need to wrapped with SwaggerValidationError
Use the `wrap_exception` method in `common.py` to wrap all possible errors.

### PR DESCRIPTION
**Pull request by prat0318**
Fixes #6 and #7

### CODE REVIEW

File: swagger_spec_validator/validator20.py [hunk 1]
```diff
@@ -5,32 +5,34 @@
...
     :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
-    :raises: :py:class:`jsonschema.exceptions.ValidationError`
     """
     log.info('Validating %s' % spec_url)
     spec_json = json.load(urllib2.urlopen(spec_url, timeout=TIMEOUT_SEC))
     validate_spec(spec_json)
 
 
-def validate_spec(spec_json):
+def validate_spec(spec_json, spec_url=None):
```
Comment (prat0318) [inline swagger_spec_validator/validator20.py]: `spec_url` is not used, but is needed to keep the interface identical to validator12's `validate_spec`.

Comment (dnephin) [inline swagger_spec_validator/validator20.py]: The python convention is to prefix unused variables with a `_` to make it clear that it's expected that this isn't used.

`_spec_url=None`

**Comment by dnephin** [comment]
lgtm (with the one variable rename)

File: swagger_spec_validator/validator20.py (hunk 1 above)
Comment (prat0318) [inline swagger_spec_validator/validator20.py]: ack.

**Pull request merged**